ert_shared/models/ensemble_experiment.py

```diff
@@ -16,6 +25,94 @@ def __init__(
     ):
         super().__init__(simulation_arguments, ert, queue_config)
 
+    async def run(self, evaluator_server_config: EvaluatorServerConfig) -> None:
+
+        # Send EXPERIMENT_STARTED
+        experiment_logger.debug("starting ensemble experiment")
+        await self.dispatch(
+            CloudEvent(
+                {
+                    "type": identifiers.EVTYPE_EXPERIMENT_STARTED,
+                    "source": f"/ert/experiment/{self.id_}",
+                    "id": str(uuid.uuid1()),
+                }
+            ),
+            0,
+        )
+
+        loop = asyncio.get_running_loop()
+        executor = concurrent.futures.ThreadPoolExecutor()
```

Comment:
Can we use the contextmanager ie: `with futures.ThreadPoolExecutor() as pool:` or otherwise we need to call `executor.shudown()` I guess.